<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QTableView" name="mainTab"/>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="1">
       <widget class="QPushButton" name="AddNewItems">
        <property name="text">
         <string>AddNewItems</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="ToLabel">
            <property name="text">
             <string>To:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="fromSpinBox"/>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="toSpinBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="FromLabel">
            <property name="text">
             <string>From:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="plusOne">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="minusOne">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Filtering expression:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>SwitchWindow</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="Delete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Filter by:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Calories</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Protein</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fat</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sodium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fiber</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Carbo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sugars</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Potass</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vitamins</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Shelf</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Weight</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cups</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rating</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="logoBtn">
        <property name="text">
         <string>About + Logo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTreeView" name="proxyView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>364</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_Table">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAddNewLines">
    <property name="title">
     <string>AddNewLines</string>
    </property>
   </widget>
   <addaction name="menuOpen_Table"/>
   <addaction name="menuAddNewLines"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
